src/dynamo/client.ts

```diff
@@ -2,19 +2,13 @@
 import dotenv from 'dotenv';
 dotenv.config();
 
-// Load the AWS SDK for Node.js
-import AWS from 'aws-sdk';
+import { DynamoDB } from '@aws-sdk/client-dynamodb';
+import { fromEnv } from '@aws-sdk/credential-providers';
 
-AWS.config.update({
-  s3ForcePathStyle: !!process.env.AWS_ENDPOINT,
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
-  region: process.env.AWS_REGION,
-});
-
-const client = new AWS.DynamoDB({
-  apiVersion: '2012-08-10',
+const client = new DynamoDB({
   endpoint: process.env.AWS_ENDPOINT || undefined,
+  credentials: fromEnv(),
```

Comment:
Since the .env has AWS_PROFILE as well as Access key/secret without setting the credentials provider it defaulted to AWS_PROFILE and trying to get from EC2 instance metadata service.

Are the servers in AWS using access key/id? If not, then I'll need to change this to a fuller credential change